Sources/Metadata.swift

```diff
@@ -68,3 +51,26 @@ struct MetadataBuilder: Builder {
                         nullable: self.nullable, example: self.example)
     }
 }
+
+extension DataType: ImmutableMappable {
+
+    public init(map: Map) {
```

Comment:
`Initializer 'init(map:)' must be declared public because it matches a requirement in public protocol 'ImmutableMappable'`